apps/io-wallet-user-func/package.json

```diff
@@ -25,17 +25,20 @@
     "@opentelemetry/semantic-conventions": "^1.27.0",
     "@pagopa/handler-kit": "^1.1.0",
     "@pagopa/handler-kit-azure-func": "^2.0.7",
+    "@pagopa/io-functions-commons": "^29.3.0",
     "@pagopa/ts-commons": "^13.1.2",
     "@peculiar/asn1-android": "^2.3.13",
     "applicationinsights": "^3.4.0",
     "asn1js": "^3.0.5",
     "cbor-x": "^1.6.0",
     "fp-ts": "^2.16.9",
     "googleapis": "^144.0.0",
+    "install": "^0.13.0",
     "io-ts": "^2.2.21",
     "io-wallet-common": "workspace:^",
     "jose": "5.9.6",
     "jsonwebtoken": "^9.0.2",
+    "npm": "^10.9.1",
```

Comment:
Why do you need `install`? Also we use `yarn` not `npm`.

Reply:
Just a typo, sometimes I forget we use yarn instead of npm! :')